atomix/pom.xml

```diff
@@ -0,0 +1,283 @@
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns="http://maven.apache.org/POM/4.0.0"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <description>Distributed systems framework.</description>
+  <modelVersion>4.0.0</modelVersion>
+  <name>Atomix Parent Pom</name>
+  <artifactId>atomix-parent</artifactId>
+  <packaging>pom</packaging>
+  <version>0.23.0-SNAPSHOT</version>
+  <inceptionYear>2020</inceptionYear>
+
+  <parent>
+    <groupId>io.zeebe</groupId>
+    <artifactId>zeebe-parent</artifactId>
+    <version>0.23.0-SNAPSHOT</version>
+    <relativePath>../parent</relativePath>
+  </parent>
+
+  <licenses>
+    <license>
+      <distribution>repo</distribution>
+      <name>Apache License, Version 2.0</name>
+      <url>http://apache.org/licenses/LICENSE-2.0</url>
+    </license>
+  </licenses>
+
+  <modules>
+    <module>utils</module>
+    <module>storage</module>
+    <module>cluster</module>
+    <module>core</module>
+  </modules>
+
+  <properties>
+    <version.java>11</version.java>
+    <!--    <license.header>https://raw.githubusercontent.com/zeebe-io/zeebe/develop/parent/COPYING-HEADER.txt</license.header>-->
+
+    <!-- Java -->
+    <classgraph.version>4.8.65</classgraph.version>
+
+    <!-- Logging -->
+    <commons.lang3.version>3.9</commons.lang3.version>
+
+    <!-- Testing -->
+    <commons.math3.version>3.6.1</commons.math3.version>
+    <concurrentunit.version>0.4.6</concurrentunit.version>
+    <config.version>1.4.0</config.version>
+
+    <!-- Core dependencies -->
+    <dockerfile.version>latest</dockerfile.version>
+    <guava.version>23.0</guava.version>
+    <jackson.version>2.10.3</jackson.version>
+    <java.version>11</java.version>
+    <jaxrs.version>2.1.1</jaxrs.version>
+    <junit.version>4.13</junit.version>
+    <jv>${project.version}</jv>
+    <kryo.version>4.0.2</kryo.version>
+    <license.header>com/mycila/maven/plugin/license/templates/APACHE-2.txt</license.header>
+    <logback.version>1.2.3</logback.version>
+
+    <!-- Maven plugins -->
+    <maven-jar-plugin.version>3.2.0</maven-jar-plugin.version>
+    <maven.assembly.plugin.version>3.2.0</maven.assembly.plugin.version>
+    <maven.clean.plugin.version>2.5</maven.clean.plugin.version>
+    <maven.dependency.plugin.version>2.7</maven.dependency.plugin.version>
+    <maven.javadoc.plugin.version>3.2.0</maven.javadoc.plugin.version>
+    <maven.source.plugin.version>3.2.1</maven.source.plugin.version>
+    <maven.surefire.plugin.version>2.22.2</maven.surefire.plugin.version>
```

Comment:
Same for all the others - we shouldn't have to specify any versions here as we should use the ones defined by the Zeebe parent